### PR TITLE
remove duplicate file

### PR DESCRIPTION
**Pull request by VickiLanger**
get_img_for_tweet.py appears to be the actual file in use

### CODE REVIEW

**Comment by VickiLanger** [comment]
Hey, @bobsany16, I think this file is just a duplicate of `get_img_for_tweet.py`. I want to get another set of eyes on it before I merge this. If you get a chance, could you glance at it and make sure I'm not trying to delete something important??

**Review by bobsany16 — APPROVED** [review]
You're good to merge! It was in fact, just a duplicate!

**Pull request merged**
